HPXMLtoOpenStudio/tests/test_schedules.rb

```diff
@@ -13,6 +13,9 @@ def setup
     @sample_files_path = File.join(@root_path, 'workflow', 'sample_files')
     @tmp_hpxml_path = File.join(@sample_files_path, 'tmp.xml')
     @tmp_schedule_file_path = File.join(@sample_files_path, 'tmp.csv')
+    @year = 2007
+    @tight_tol = 0.005
+    @stochastic_sch_tol = 0.075
```

Comment:
Help me understand, why do we need a higher tolerance for stochastic schedules? 7.5% is pretty high.